qemu/tests/cfg/blockdev_inc_backup_migrate_without_bitmap.cfg

```diff
@@ -47,6 +49,7 @@
         - persistent:
             bitmap_debugged = yes
             full_backup_options = {"sync": "full", "persistent": "on"}
+            no ceph
```

Comment:
The same concern as above, do we need to add a FIXME here for our future update when the qcow2 ceph issue is fixed

Reply:
Have updated, thanks zhenchao.